### PR TITLE
Add more visual separation between builds, add copy to clipboard for webhook URLs

### PR DESCRIPTION
**Pull request by jwforres**
Copy to clipboard using ZeroClipboard, only works in browser's with Flash support (otherwise button will not appear).

updated-builds-page

show_webhook_url

### CODE REVIEW

**Comment by jwforres** [comment]
@liggitt review
@bparees @danmcp  FYI

**Comment by bparees** [comment]
+1, thanks @jwforres !

File: assets/app/styles/_core.less [hunk 1]
```diff
@@ -17,6 +17,10 @@ html, body {
   .container-main {
     background-color: @body-bg;
     .flex(@columns: 1);
+
+    hr {
```
Comment (liggitt) [inline assets/app/styles/_core.less]: remove unused style

File: assets/app/views/builds.html [hunk 2]
```diff
@@ -8,58 +8,69 @@
...
+          <div>
+            Triggers: 
+            <ul>
+              <li ng-repeat="trigger in buildConfig.triggers">
+                <span ng-switch="trigger.type">
+                  <span ng-switch-when="github">
+                    <span style="margin-right: 5px;">GitHub webhook URL</span>
+                    <a href="{{'webhooks' | helpLink}}" target="_blank"><i class="pficon pficon-help"></i></a>
+                    <div>
+                      <span click-to-reveal link-text='"Show URL..."' style="margin-right: 5px;">{{buildConfigName | webhookURL : trigger.type : trigger.github.secret : project.metadata.name}}</span>
```
Comment (liggitt) [inline assets/app/views/builds.html]: Change directive to evaluate link-text as a string

File: assets/app/views/builds.html [hunk 3]
```diff
@@ -8,58 +8,69 @@
...
+                      <copy-to-clipboard-button clipboard-text="buildConfigName | webhookURL : trigger.type : trigger.github.secret : project.metadata.name"></copy-to-clipboard-button>
+                    </div>
+                  </span>
+                  <span ng-switch-when="generic">
+                    <span style="margin-right: 5px;">Generic webhook URL</span>
+                    <a href="{{'webhooks' | helpLink}}" target="_blank"><i class="pficon pficon-help"></i></a>
+                    <div>
+                      <!-- <input class="small " readonly="readonly" value="{{buildConfigName | webhookURL : trigger.type : trigger.generic.secret : project.metadata.name}}" select-on-focus style="width: 100%; display: none;"> -->
```
Comment (liggitt) [inline assets/app/views/builds.html]: remove unused

**Comment by liggitt** [comment]
LGTM

**Comment by jwforres** [comment]
[merge]

**Pull request merged**